editor/src/components/canvas/__snapshots__/ui-jsx-canvas.spec.tsx.snap

```diff
@@ -21228,9 +21710,15 @@ exports[`UiJsxCanvas render renders fine with two components that reference each
       width: 375px;
       height: 812px;
     \\"
+    data-paths=\\":utopia-storyboard-uid\\"
     data-uid=\\"utopia-storyboard-uid\\"
   >
-    <div data-uid=\\"2a2~~~1 aaa bbb BBB scene\\">great</div>
+    <div
+      data-uid=\\"2a2~~~1 aaa bbb BBB scene\\"
+      data-paths=\\":2a2~~~1 utopia-storyboard-uid/scene:BBB :utopia-storyboard-uid/scene\\"
+    >
+      great
+    </div>
```

Comment:
I'm not sure what has happened with the paths here, but something is definitely wrong :grimacing:

Reply:
I would say this is fine as is, :2a2~~~1 is an irrelevant path, and utopia-storyboard-uid/scene:BBB is valid. aaa and bbb come from arbitrary JS blocks so we can't edit them hence the missing inner template paths